### PR TITLE
Fixes #789 - RAF Akrotiri (LCRA) Profiles & Displays

### LINKED ISSUE #789: Incorporate RAF Akrotiri (LCRA) Profiles & Displays
## What is the current situation?
Not Included

## What is the proposed change?
Include

## Reference materials (e.g. screenshots)
https://community.vatsim.uk/files/downloads/file/305-raf-akrotiri-lcra-controller-pack-add-on/

### PR DESCRIPTION
**Pull request by luke11brown**
Fixes #789 

# Summary of changes

Added Profiles & Displays for RAF Akrotiri (LCRA)
Uses same symbology as Leeds as it's based off the same system (Cobham)
Uses separate sector file for obvious reasons

Investigation needed:

- [x] UKCP squawk allocation?
- [ ] UKCP initial altitudes to be setup?
- [ ] UKCP min stack level?
- [x] @19wintersp - SCT2 script may need to ignore some files added here
- [ ] Add requisite danger areas to TopSky?

# Screenshots (if necessary)
image

image

### CODE REVIEW

**Comment by 19wintersp** [comment]
I should be able to write up the required changes later today; easiest option would be just to write up a manual ignore for ASRs included by PRFs in the Military folder, but it wouldn't require too much effort to extend the functionality to all SMRs by selecting the relevant sectorfile based on the PRF. Unfortunately ES doesn't seem to want to include the `SECTORFILE:` line in the SMR ASRs even though it is in the Radar ASRs for some reason...? Can be worked around though

File: UK/Data/Sector/Akrotiri LCRA.ese [hunk 1]
```diff
@@ -0,0 +1,272 @@
+[POSITIONS]
+;Akrotiri
+LCRA_APP:Akrotiri Radar:123.600:RAR::LCRA:APP:::1260:1277
```
Comment (luke11brown) [inline UK/Data/Sector/Akrotiri LCRA.ese:3]: Change to "Approach"

File: UK/Data/Sector/Akrotiri LCRA.ese [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+LCRA_P_APP:Akrotiri Talkdown:125.700:RAP::LCRA:APP:::1260:1277
+LCRA_TWR:Akrotiri Tower:130.075:RAT::LCRA:TWR:::1260:1277
+LCRA_GND:Akrotiri Ground:122.100:RAG::LCRA:GND:::1260:1277
+LCRA_ATIS:AKrotiri Information:125.000:RAI::LCRA:ATIS::::
+
+;Nicosia
+EURS_FSS:Eurocontrol:135.550:ES::EURS:FSS:::0500:0577:N041.04.56.636:E011.44.48.111:N037.39.17.755:E017.28.14.655:N037.19.36.372:E029.52.50.319:N037.10.51.314:E039.07.24.821
```
Comment (luke11brown) [inline UK/Data/Sector/Akrotiri LCRA.ese:10]: EURS still valid?

File: UK/Data/Sector/Akrotiri LCRA.ese [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+SID:LCRA:28:EAST3:DE28 IREFA
+SID:LCRA:28:WEST1:DE28 MEZUS
+SID:LCRA:28:WEST2:DE28 ANANE
+STAR:LCRA:10:ANANEA:ANANE R214N R102N AKR11
+STAR:LCRA:10:ANANEB:ANANE R214K R131K
+STAR:LCRA:10:IREFAA:IREFA R111N R102N AKR11
+STAR:LCRA:10:IREFAB:IREFA R111O IAK97
+STAR:LCRA:10:MEZUSA:MEZUS R253N R102N AKR11
+STAR:LCRA:10:MEZUSB:MEZUS R253K R131K
+STAR:LCRA:28:IREFAB:IREFA R111O IAK97 IAK46
```
Comment (luke11brown) [inline UK/Data/Sector/Akrotiri LCRA.ese:68]: check radials

File: UK/Data/Settings/Local/Akrotiri/Profiles.txt [hunk 4]
```diff
@@ -0,0 +1,18 @@
+PROFILE
+PROFILE:LCRA_APP:150:5
+ATIS2:"Akrotiri Radar"
```
Comment (luke11brown) [inline UK/Data/Settings/Local/Akrotiri/Profiles.txt:3]: Change to "Approach"

File: UK/Data/Settings/VCCS_Akrotiri.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+VCCS
+Ts3Server:ukvccs.vamsys.io
+Ts3Port:9988
+Ts3G2GChannel:LL:GMP:16777215:0:LLD
+Ts3G2GChannel:LL:PLN:16777215:1:LLP
+Ts3G2GChannel:LL:GMC 1:16777215:3:LL1
+Ts3G2GChannel:LL:GMC 2:16777215:4:LLG
```
Comment (luke11brown) [inline UK/Data/Settings/VCCS_Akrotiri.txt:7]: Dummy frequencies, to be amended.

File: UK/Data/Sector/Akrotiri LCRA.ese [hunk 6]
```diff
@@ -0,0 +1,272 @@
+[POSITIONS]
+;Akrotiri
+LCRA_APP:Akrotiri Radar:123.600:RAR::LCRA:APP:::1260:1277
+LCRA_P_APP:Akrotiri Talkdown:125.700:RAP::LCRA:APP:::1260:1277
+LCRA_TWR:Akrotiri Tower:130.075:RAT::LCRA:TWR:::1260:1277
+LCRA_GND:Akrotiri Ground:122.100:RAG::LCRA:GND:::1260:1277
+LCRA_ATIS:AKrotiri Information:125.000:RAI::LCRA:ATIS::::
```
Comment (19wintersp) [inline UK/Data/Sector/Akrotiri LCRA.ese:7]:
```suggestion
LCRA_ATIS:Akrotiri Information:125.000:RAI::LCRA:ATIS::::
```

File: UK/Data/ASR/Akrotiri/Akrotiri Radar.asr [hunk 7]
```diff
@@ -0,0 +1,129 @@
+DisplayTypeName:Standard ES radar screen
+DisplayTypeNeedRadarContent:1
+DisplayTypeGeoReferenced:1
+SECTORFILE:UK\Data\Sector\Akrotiri LCRA.sct
+SECTORTITLE:Akrotiri LCRA.sct
```
Comment (luke11brown) [inline UK/Data/ASR/Akrotiri/Akrotiri Radar.asr:5]: Remove

**Pull request merged**